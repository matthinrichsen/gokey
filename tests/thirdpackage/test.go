package thirdpackage

import (
	"github.com/matthinrichsen/gokey/tests/anotherPackage"
)

var _ = anotherPackage.LastStruct{1}
