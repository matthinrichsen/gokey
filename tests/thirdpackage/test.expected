package thirdpackage

import (
	"github.com/matthinrichsen/gokey/tests/anotherPackage"
)

var _ = anotherPackage.LastStruct{Int: 1}
